sympy/integrals/rubi/utility_function.py

```diff
@@ -258,7 +261,7 @@ def IntegersQ(*var):
     return all(IntegerQ(i) for i in var)
 
 def ComplexNumberQ(*var):
```

Comment:
`*var` -> `*vars`.

Reply:
Many functions has used "*var". So should I change all of them to "*vars" .And, also in function body.

Reply:
No, please ignore making this change.